src/commands/helper.py

```diff
@@ -34,15 +34,32 @@ def create_logging_embed(interaction: discord.Interaction, **kwargs):
 def create_bot_errors(bot: Bot) -> None:
     @bot.tree.error
     async def on_app_command_error(interaction: discord.Interaction, error):
-        # Check if the error is due to a cooldown
+        # Handle CommandOnCooldown error
         if isinstance(error, discord.app_commands.CommandOnCooldown):
             remaining_time = int(error.retry_after) + int(time.time())
             await interaction.response.send_message(
-                f"This command is on cooldown. Time remaining: <t:{remaining_time}:R>",
+                f"This command is on cooldown. Time remaining: <t:{remaining_time}:R>.",
```

Comment:
i think we can change this text to something like "Try again in" instead of "Time remaining"